### PR TITLE
 AI Linear Activation Function (Issue #2700)

### PR DESCRIPTION
**Pull request by shanmukha-m-k**
AI Linear Activation Function (Issue #2700)

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [*] All writings are my own.
- [*] My entry follows the Codecademy Docs style guide.
- [*] My changes generate no new warnings.
- [*] I have performed a self-review of my own writing and code.
- [*] I have checked my entry and corrected any misspellings.
- [*] I have made corresponding changes to the documentation if needed.
- [*] I have confirmed my changes are not being pushed from my forked `main` branch.
- [*] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [*] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+Title: 'Linear Activation Function'
+Description: 'The linear activation function also known as identity function, where the activation is proportional to the input.'
```
Comment (SSwiniarski) [inline content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md:3]:
```suggestion
Description: 'The linear activation function, also known as the identity function, is where the activation is proportional to the input.'
```

File: content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  - 'AI'
+  - 'Machine Learning'
+  - 'Neural Networks'
+  - 'Functions'
+CatalogContent:
+  - 'machine-learning'
+  - 'path/data-science'
+---
+
+The **linear activation function** is also known as **straight-line function** where the activation is proportional to the input.The function doesn't do anything to the weighted sum of the input, it simply spits out the value it was given.
```
Comment (SSwiniarski) [inline content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md:18]:
```suggestion
The **linear activation function**, also known as the identity function, is where the activation is proportional to the input. The function doesn't do anything to the weighted sum of the input, it simply spits out the value it was given.
```

File: content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+Mathematically, it can be defined as:
+
+![linear Activation Function Equation](https://raw.githubusercontent.com/Codecademy/docs/main/media/linear-equation.png)
+
+The value of variable a can be any constant value
+chosen by the user.
```
Comment (SSwiniarski) [inline content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md:25]: There's no variable "a" in the above equation.

File: content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+### Usage and Limitations
+
+The problem with this activation is that it cannot be defined in a specific range. Applying this function in all the nodes makes the activation function work like linear regression.
```
Comment (SSwiniarski) [inline content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md:29]:
```suggestion
The problem with this activation function is that it cannot be defined for a specific range. Applying this function in all the nodes makes the activation function work like linear regression.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@shanmukha-m-k I did a first review and left a few comments.

**Comment by shanmukha-m-k** [comment]
i have commited suggested changes plz review once

**Review by SSwiniarski — APPROVED** [review]
@shanmukha-m-k Looks good for a second review.

**Review by HishamT — CHANGES_REQUESTED** [review]
Hey @shanmukha-m-k, I have reviewed your entry and left you some comments. 

One major issue I noticed is that some of text and a picture were taken from another article on the web. As this can be seen as plagiarism, you will have to rewrite those parts. Remember, that all writing submitted must be original! We do not allow plagiarized content on Codecademy. Thank you!

**Comment by shanmukha-m-k** [comment]
@HishamT Thanks for your suggestion sir  . I committed suggested changes please review once

File: content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  - 'AI'
+  - 'Machine Learning'
+  - 'Neural Networks'
+  - 'Functions'
+CatalogContent:
+  - 'machine-learning'
+  - 'path/data-science'****
+---
+
+The **linear activation function**, also known as the identity function, is one of the most straight-forward activation function,where the output of linear activation function is exactly same as input.
```
Comment (HishamT) [inline content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md:18]:
```suggestion
The **linear activation function**, also known as the identity function, is a straight-forward activation functions where the output is exactly same as the input.
```

File: content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+CatalogContent:
+  - 'machine-learning'
+  - 'path/data-science'
+---
+
+The **linear activation function**, also known as the identity function, is where the activation is proportional to the input. The function doesn't do anything to the weighted sum of the input, it simply spits out the value it was given.
+
+Mathematically, it can be defined as:
+
+![linear Activation Function Equation](https://raw.githubusercontent.com/Codecademy/docs/main/media/linear-equation.png)
```
[the remark this answers is not in the document]
Comment (HishamT) [reply content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md:24]: This link yields a 404 error

File: content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+  - 'Machine Learning'
+  - 'Neural Networks'
+  - 'Functions'
+CatalogContent:
+  - 'machine-learning'
+  - 'path/data-science'****
+---
+
+The **linear activation function**, also known as the identity function, is one of the most straight-forward activation function,where the output of linear activation function is exactly same as input.
+In general , linear activation function simply add weighted sum with bias and pass it as output.
```
Comment (HishamT) [inline content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md:19]: This statement contains some grammatical errors and is unclear. Can you reword it?

File: content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+where :
+
+![linear Activation Function](https://raw.githubusercontent.com/Codecademy/docs/main/media/linear-equation-express.png)
+
+### Usage and Limitations
+
+The problem with this activation function is that it is limited in the context of deep neural network's.
+
+Using linear activation functions would not able to the model to learn complex non-linear relationships between input's and output's.
```
Comment (HishamT) [inline content/ai/concepts/neural-networks/terms/linear-activation-function/linear-activation-function.md:34]: This statement needs to be reworded.

**Review by HishamT — CHANGES_REQUESTED** [review]
Hey @shanmukha-m-k, I have reviewed your entry and left you some comments.

**Comment by shanmukha-m-k** [comment]
@SSwiniarski @HishamT  Hi sir , I have committed suggested changes please review once

**Comment by SSwiniarski** [comment]
@HishamT Are we good to merge?

**Review by HishamT — APPROVED** [review]
LGTM

**Pull request merged**